srcalc/src/main/webapp/WEB-INF/views/fragments/discreteNumericalFragment.jsp

```diff
@@ -0,0 +1,18 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
```

Comment:
`discreteNumericalFragment` is actually a little ambiguous. How is this fragment different from `discreteNumericalProperties.jspf`? Can you think of a less ambiguous name?